azure-pipelines/ci.yml

```diff
@@ -38,16 +38,36 @@ jobs:
           versionSpec: 3.6
       - script: 'npm run build'
         displayName: 'npm build'
-      - script: 'npm test'
-        displayName: 'npm test'
       - script: 'npm pack'
         displayName: 'npm pack'
       - task: PublishBuildArtifacts@1
         inputs:
           PathtoPublish: '$(VAR_BUILD_ARTIFACT_STAGING_DIRECTORY)'
           ArtifactName: '$(VAR_ARIFACT_NAME)'
           publishLocation: 'Container'
-
+          
+  - job: TestReport
+    pool:
+      vmImage: 'Ubuntu 16.04'
+      pythonVersion: '3.6'
+    steps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: '$(NodeVersion)'
+        displayName: 'Install Node.js $(NodeVersion)'
+      - task: Npm@1
+        displayName: 'npm install'
+        inputs:
+          verbose: false
+      - task: UsePythonVersion@0
+        displayName: 'Use Python 3.6'
+        inputs:
+          versionSpec: 3.6
+      - script: 'npm run build'
+        displayName: 'npm build'
+      - script: 'npm run test-report'
```

Comment:
why do we need a new job for this?  can this be put into the original place?

Reply:
I just thought it's not very good if we put the test step inside the build step.

Reply:
If test report failed, should we stop the 'npm pack'?

Reply:
npm pack just a step to pack this code, it won't affect anything.